docs: vacuum-module-instruction-manual (#21157)
OpenTrons/opentrons-api: docs/vacuum/docs/installation.md

---
title: "Vacuum Module: Installation"
description: "Installing the Vacuum Module on a Flex robot."
---

Follow these instructions to install the Vacuum Module.

!!! warning
    Turn off the power to the Flex before beginning the installation process. This prevents the robot from operating unexpectedly during setup.

<div class="instruction-list" markdown>

## Part 1: Deck preparation

1. Clear the deck of all modules and labware to give yourself sufficient room to work.

2. Remove the waste bin (if installed) and any modules, labware, or deck adapters and/or plates from slots A3—A4.

    <figure markdown>
    ![Deck map with sots A3 and A4 highlighted](images/deck-map.png)
    <figcaption>Vacuum Module deck map location.</figcaption>
    </figure>

3. Install the Vacuum Module deck adapter in slots A3–A4 only. Fasten it to the deck with the original deck screws or use the screws provided with the module.

> 4. Set the manifold base plate on the deck adapter in slot A-3.

Reviewer: what is the base plate? didn't see it labeled in products and specs page. I assume it's the same as the manifold base, but the different term threw me

Author: Fixing inconsistent use and replacing with "vacuum base."

![Manifold fitting disconnected](images/fitting-disconnected.svg)
![Manifold fitting connected](images/fitting-connected.svg)
</figure>

6. Run the hose from the manifold through the notch on the side of the deck plate adapter and into the space below the deck. The area below the deck keeps cables, tubes, hoses, and other module connections from cluttering the work area.

    <figure markdown>
    ![Deck plate with magnified image of vacuum hose notch](images/deck-plate-notch.png){ width="80%" }
    <figcaption>Adapter notch provides below-deck access for the vacuum hose.</figcaption>
    </figure>

7.  Remove one of the lower cosmetic side panels on the robot and bring the remaining hose section through this opening. From here, you can connect the hose to the carboy.

    <figure class="screenshot" markdown>
    ![Flex with a single lower cosmetic panel removed. Shows vacuum hose exiting robot.](images/side-panel.png)
    <figcaption>Vacuum hose exiting Flex from a lower side panel.</figcaption>
    </figure>

## Part 2: External components setup

6. Position the carboy cradle and the 2-liter glass carboy on the lab bench at or below the level of the Control Box.

> 7. Place the cap on the carboy and hand tighten it. Do not over-tighten the cap, this is not a trial of strength.

Reviewer: ```suggestion
7. Place the cap on the carboy and hand tighten it. Do not over-tighten the cap; this is not a trial of strength.
```

not a trial of strength ❤️

8. Attach the free end of the 6 mm (&frac14;") vacuum tube coupling insert into the connector on the cap. Measure and cut the tubing to length if necessary.

> 9. Attach the 9.5 mm (&frac38;") vacuum hose and coupling inserts into the connector on the cap and on the Control Box. Measure and cut the hose to length if necessary.

Reviewer: can users order additional hoses? if they cut it to a certain length that ends up not working after a lab move, etc.

Author: They can probably re-order from Opentrons or get their own. The specs page show the inner diameter (ID) and material if they want to get more hose. The hose can be made of anything as long as it matches the ID.

---
title: "Vacuum Module: Installation"
description: "Installing the Vacuum Module on a Flex robot."
---

Follow these instructions to install the Vacuum Module.

!!! warning
    Turn off the power to the Flex before beginning the installation process. This prevents the robot from operating unexpectedly during setup.

<div class="instruction-list" markdown>

## Part 1: Deck components

1. Clear the deck of all modules and labware to give yourself sufficient room to work.

> 2. Remove the waste bin (if installed) and any modules, labware, or deck adapters and/or plates from slots A3–A4.

Reviewer 2: ```suggestion
2. Remove the trash bin (if installed) and any modules, labware, deck adapters, or plates from slots A3–A4.
```

    <figure markdown>
    ![Deck map with sots A3 and A4 highlighted](images/deck-map.png)
    <figcaption>The Vacuum Module installs in slots A3–A4.</figcaption>
    </figure>

3. Install the Vacuum Module deck adapter in slots A3–A4 only. Fasten the adapter to the deck with the original screws or use the screws provided with the module.

> 4. Set the vacuum base on the deck adapter in slot A-3.

Reviewer 2: ```suggestion
4. Set the vacuum base on the deck adapter in slot A3.
```
